lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_builtin_method.rb

```diff
@@ -10,10 +10,12 @@ class MiqAeBuiltinMethod
       vm_retire_task
       service_retire_task
       orchestration_stack_retire_task
+      miq_provision_task
```

Comment:
As discussed on gitter I've now improved this part: `PhysicalServerProvisionTask` now resolves to `physical_server_provision_task` so that it's more obvious what we're provisioning. This also simplifies the IF statements below because task type is now unique.